configs/server/forseti_conf_server.yaml.in

```diff
@@ -289,3 +289,11 @@ notifier:
         enabled: false
         # gcs_path should begin with "gs://"
         gcs_path:
+
+    inventory:
+      summary:
+        enabled: false
+        # gcs_path should begin with "gs://"
+        # data_format may be one of: csv (the default) or json
+        data_format: csv
+        gcs_path: gs://{INV_SUMMARY_BUCKET}/inv_summary
```

Comment:
What if someone wants to get this by slack or email?  How would we add those here?  If you are looking at the CSCC, that's very specific for CSCC, in that there will only be one type of notification.

Reply:
I understood that a GCS upload is the only option we support for inventory summary notifications. If we *do* decide to support additional channels, the configuration should be generalized to look something like

    inventory:
      summary:
        enabled: false
        notifiers:
          # Upload inventory summary to GCS.
          - name: gcs_inv_summary
            configuration:
              data_format: json
              gcs_path: gs://{INV_SUMMARY_BUCKET}/inv_summary
          # Email inventory summary
          - name: email_inv_summary
            configuration:
              sendgrid_api_key: {SENDGRID_API_KEY}
               sender: {EMAIL_SENDER}
               recipient: {EMAIL_RECIPIENT}